setup.cfg

```diff
@@ -49,3 +49,10 @@ test = pytest
 
 [tool:pytest]
 addopts = --pycodestyle -v --cov --cov-report term-missing
+
+[tool:isort]
+# https://github.com/timothycrosley/isort
+multi_line_output = 4
+line_length = 72
+known_first_party =
+    piplicenses
```

Comment:
@cdce8p Thank you for your great contribution. This is a very useful tool.

I have something to ask for future reference. Why do you want to overwrite the `line_length` setting from 79 (as default) to 72? I'd like to hear if there is any reason to recommend it.

Reply:
To be honest, I don't really know. It's something I have picked up somewhere over time and it seems to work great. I guess an advantage would be that with only `72` characters not to many `from` imports are on one line, see `test_piplicenses.py` for example.

Regarding the default `79`: Personally I would recommend to change it. I tend to use `119` for my projects. That doesn't mean that every line should now be longer, but that in some **limited** cases it improves readability when you otherwise would be only a few characters over the limit. In fairness a line that approaches 119 would probably be best split up anyway, but it's our choice if it's really the best option, not of the style checker.

Reply:
> To be honest, I don't really know. It's something I have picked up somewhere over time and it seems to work great. I guess an advantage would be that with only `72` characters not to many `from` imports are on one line, see `test_piplicenses.py` for example.

Okay, I understand. Indeed, it seems to be working well.

> That doesn't mean that every line should now be longer, but that in some **limited** cases it improves readability when you otherwise would be only a few characters over the limit. 

In limited cases, you're right. I'm following the Python community standard for now.